docs/supporting/requirements.md

```diff
@@ -22,7 +22,7 @@ We recommend:
 VS Code is supported on the following platforms:
 
 * Windows 10 and 11 (32-bit and 64-bit)
-* OS X High Sierra (10.13+)
+* macOS versions receiving security fixes from Apple, this typically includes the latest release and the 2 versions before it
```

Comment:
How about: 
"macOS versions with Apple security update support. This is typically the latest release and the two previous versions."

Reply:
Sounds good 👍 

```suggestion
* macOS versions with Apple security update support. This is typically the latest release and the two previous versions.
```